cfme/infrastructure/cluster.py

```diff
@@ -143,6 +144,14 @@ def is_displayed(self):
     entities = View.nested(ClusterDetailsEntities)
 
 
+class ClusterAllVMsView(VmsTemplatesAllView):
+    """The view is used by test_ssa_cluster_relationships"""
```

Comment:
Can you make the description more generic?

Reply:
Done